src/Yasumi/data/translations/ascensionDay.php

```diff
@@ -19,5 +19,6 @@
     'da_DK' => 'Kristi Himmelfartsdag',
     'nb_NO' => 'Kristi himmelfartsdag',
     'sv_SE' => 'Kristi himmelsfärds dag',
-    'fi_FI' => 'Helatorstai'
+    'fi_FI' => 'Helatorstai',
+    'de'    => 'Christi Himmelfahrt'
```

Comment:
Although 'de' is an official locale, it is better to use a locale that identifies also the region as well. In this case 'de_DE' is preferred. Can you adjust your PR accordingly?